app/models/document/document_filter.rb

```diff
@@ -71,15 +71,18 @@ def to_date
   end
 
   def unpaginated_documents
-    documents = Document
-    documents = documents.where(block_type: Schema.valid_schemas)
+    documents = Document.where(block_type: Schema.valid_schemas)
+                         .joins(:editions)
+                         .merge(Edition.most_recent_for_document)
+                         .merge(Edition.active)
```

Comment:
I think we need to update the document filter tests here. Intrigued as to why they haven't failed actually 🤔